js/src/components/modals/KeyphrasesTable.js

```diff
@@ -127,7 +127,7 @@ class KeyphrasesTable extends Component {
 									</span>
 								</HelpLink>
 							</th>
-							{ renderAction && <td /> }
+							{ renderAction && <td className="yoast-table--nobreak" /> }
```

Comment:
Shouldn't the class be added to the `<td>` in line 160 as well? Here it seems to me that it just sets it for the empty cell in the table header, and not for the body cells that host the button...